source/change-streams/change-streams.rst

```diff
@@ -9,8 +9,8 @@ Change Streams
 :Status: Accepted
 :Type: Standards
 :Minimum Server Version: 3.6
-:Last Modified: April 3, 2019
-:Version: 1.7.0
+:Last Modified: Febrary 10, 2020
+:Version: 1.7.1
```

Comment:
I wasn't sure what constitutes a minor vs patch version bump. Should this be 1.7.1 or 1.8.0?

Reply:
IMO 1.8.0 seems warranted. This is a new behavior change. That being said I often forget to update the version...